libsolidity/formal/CHC.h

```diff
@@ -147,6 +149,10 @@ class CHC: public SMTEncoder
 	/// @returns true if _function is Natspec annotated to be abstracted by
 	/// nondeterministic values.
 	bool abstractAsNondet(FunctionDefinition const& _function);
+
+	/// @returns true if external calls should be considered trusted.
+	/// If that's the case, their code is used if available at compile time.
+	bool externalCallsIsTrustedMode();
```

Comment:
```suggestion
	bool externalCallsAreTrustedMode();
```
:-)?

Reply:
;$

Reply:
the original one was more like "external calls encoding is trusted mode", so the verb isn't really applied to `external calls`. How about "encodeExternalCallsAsTusted"?